plugins/module_utils/gpg_utils.py

```diff
@@ -162,9 +162,12 @@ def get_recipients_from_encrypted_file(self, slug) -> List[str]:
             with open(file, "rb") as f:
                 recipient_subkeys = self.__gpg.get_recipients(f.read())
             for recipient_subkey in recipient_subkeys:
-                recipients.append(
-                    self.__gpg.list_keys(keys=recipient_subkey).fingerprints[0]
-                )
+                try:
+                    recipients.append(
+                        self.__gpg.list_keys(keys=recipient_subkey).fingerprints[0]
+                    )
+                except IndexError:
+                    raise GPGException(f"Subkey {recipient_subkey} has no known associated keys")
```

Comment:
This error message isn't really good IMO. I'd suggest `Can't find a primary key in your keyring for encryption subkey {recipient_subkey}` or something like that.

Aside of that: This is something which crashed the module before, so this is a `fix`, not a `chore`. Just to make sure: The GPGException is caught and handled later on, right? This doesn't crash the module anymore, right?